components/card.css

```diff
@@ -0,0 +1,95 @@
+.card,
+.card-horizontal {
+    border: 1px solid var(--theme300);
+    padding: 1rem;
+    line-height: 1.5;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--theme50);
+    border-top-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    position: relative;
+    min-width: 15rem;
+}
+
+.card {
+    max-width: 20rem;
+    flex-direction: column;
+}
+
+.card-horizontal {
+    justify-content: space-between;
+    max-width:35rem;
+}
+
+.card-img {
+    width: 100%;
+    min-width: 15rem;
+    height:30vh;
+    border-radius: 1rem;
+    align-self: center;
+}
+
+.card-shadow {
+    box-shadow: 20px 20px 10px var(--grey);
+}
+
+
+.card-badge {
+    position: absolute;
+    top: 8%;
+    left: 5%;
+    padding: 2px;
+    line-height: initial;
+    border-radius: unset;
+    font-size: 0.7em;
+}
+
+.card-badge:after {
+    content: "";
+    position: absolute;
+    left: 100%;
+    top: 0;
+    border-width: 10px 5px;
```

Comment:
You can use border shorthand property.

Reply:
Thanks for previewing. Here specifically width needed to change that's why used this